src/components/data/SavedColumnSettings.js

```diff
@@ -52,12 +52,24 @@ const useSavedColumnSettings = ({ workspaceId, snapshotName, entityMetadata, ent
 
   const signal = useCancellation()
 
+  // Column settings are now stored as JSON in the rawls database (max size 16mb)
+  // In an earlier version of this feature, they were stored as Strings in rawls (max size 64kb)
+  // For backwards-compatibility purposes, we need to support both cases. So we will try to parse
+  // the column settings as a String and if that fails then we will parse it as regular JSON
+  const parseAsStringOrJson = attr => {
+    try {
+      return JSON.parse(attr)
+    } catch (e) {
+      return attr
+    }
+  }
```

Comment:
`JSON.parse()` will succeed for inputs of numbers, booleans - does this matter to you?

Reply:
I don't think so. If a user sets `columnSettings` to a number or boolean, the UI handles it gracefully